packages/material-ui/src/Chip/Chip.d.ts

```diff
@@ -4,15 +4,55 @@ import { OverridableComponent, OverrideProps } from '../OverridableComponent';
 
 export interface ChipTypeMap<P = {}, D extends React.ElementType = 'div'> {
   props: P & {
+    /**
+     * Avatar element.
+     */
     avatar?: React.ReactElement;
+    /**
+     * This prop isn't supported.
+     * Use the `component` prop if you need to change the children structure.
+     */
+    children?: null;
```

Comment:
```suggestion
    children?: never;
```
would've been better but throws during `yarn proptypes` because there's no sensible proptype for it. This works and can be merged. We can improve later.